.gitignore

```diff
@@ -0,0 +1,7 @@
+**/*/node
+**/*/node_modules
+**/*/package.json
+**/*/package-lock.json
+**/*/target
```

Comment:
Can't we just put `target/` etc here to declare files that might be at any path? I know for example that pom.xml.versionsBackup could be at any path, but that doesnt have the wildcard prefix